service/storage_service.cc

```diff
@@ -1899,6 +2035,17 @@ future<std::unordered_set<dht::token>> storage_service::get_local_tokens() {
     });
 }
 
+future<db_clock::time_point> storage_service::get_local_streams_timestamp() {
```

Comment:
s/get_local_streams_timestamp/get_local_cdc_streams_timestamp
Move to cdc.cc?

Reply:
Moved to cdc:: namespace and cdc/generation module